schema.graphql

```diff
@@ -1,11 +1,14 @@
 type TokenSale @entity {
   id: ID!
+  bidNonce: BigInt!
   tokenId: String!
   contractAddress: String!
   amount: BigInt!
   buyer: Account!
   seller: Account!
   timestamp: BigInt!
+  paymentToken: String
+  topLevelDomainId: String
```

Comment:
Nit: Is this needed? We can calculate this through other means

Reply:
https://thegraph.com/docs/en/developer/assemblyscript-api/#access-to-smart-contract-state